src/zhinst/toolkit/driver/devices/pqsc.py

```diff
@@ -113,7 +113,7 @@ def wait_done(self, *, timeout: float = 10, sleep_time: float = 0.005) -> None:
         except TimeoutError as error:
             raise TimeoutError("PQSC timed out.") from error
 
-    def check_ref_clock(self, *, timeout: int = 30, sleep_time: int = 1) -> bool:
+    def check_ref_clock(self, *, timeout: float = 30, sleep_time: float = 1) -> bool:
```

Comment:
The default value must match the typehint.
```suggestion
    def check_ref_clock(self, *, timeout: float = 30.0, sleep_time: float = 1.0) -> bool:
```

Reply:
done